Components/Utilities/utilities.html

```diff
@@ -0,0 +1,186 @@
+                    </li>
+                </ul>
+            </nav>
+        </aside>
+        <!--Sidebar ends here-->
+        <div class="component-container">
+            <div class="heading-container">
+                <div class="examples-description">
+                    <h1 class="h1-size-1">Text Utilities</h1>
+                    <p class="p-description">
+                        Text Utilities are various predefined styles that can be used to
+                        style,align and use them instantly
+                    </p>
+                    <h1 class="h1-size-1">Headings</h1>
+                    <p class="p-description">
+                        To use heading styles, just the add class name of the desired font
+                        size to any h tags.
+                    </p>
+                </div>
+
+                <div class="examples-component flex-center row-gap-1">
+                    <h1 class="h1-size-1">h1. This is a very large header.</h1>
+                    <h2 class="h2-size-1">h2. This is a large header.</h2>
+                    <h3 class="h3-size-1">h3. This is a medium header.</h3>
+                    <h4 class="h4-size-1">h4. This is a moderate header.</h4>
+                    <h5 class="h5-size-1">h5. This is a small header.</h5>
+                    <h6 class="h6-size-1">h6. This is a tiny header.</h6>
+                </div>
+                <div class="examples-code">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%2520class%253D%2522h1-size-1%2522%253Eh1.%2520This%2520is%2520a%2520very%2520large%2520header.%253C%252Fh1%253E%250A%253Ch2%2520class%253D%2522h2-size-1%2522%253Eh2.%2520This%2520is%2520a%2520large%2520header.%253C%252Fh2%253E%250A%253Ch3%2520class%253D%2522h3-size-1%2522%253Eh3.%2520This%2520is%2520a%2520medium%2520header.%253C%252Fh3%253E%250A%253Ch4%2520class%253D%2522h4-size-1%2522%253Eh4.%2520This%2520is%2520a%2520moderate%2520header.%253C%252Fh4%253E%250A%253Ch5%2520class%253D%2522h5-size-1%2522%253Eh5.%2520This%2520is%2520a%2520small%2520header.%253C%252Fh5%253E%250A%253Ch6%2520class%253D%2522h6-size-1%2522%253Eh6.%2520This%2520is%2520a%2520tiny%2520header.%253C%252Fh6%253E"
+                        style="
+                width: 631px;
+                height: 282px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              " sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </div>
+
+            <div class="text-alignment-container flex-col-start">
+                <div class="examples-description" style="margin-top: 0.5rem">
```

Comment:
Try to avoid inline styling. It might cause issues while further development of the code.